library/build.gradle

```diff
@@ -20,7 +20,7 @@ dependencies {
     compile project(':third_party:disklrucache')
     compile project(':annotation')
     compile "com.android.support:support-v4:${SUPPORT_V4_VERSION}"
-
+    compile "com.android.support:appcompat-v7:${SUPPORT_V7_VERSION}"
```

Comment:
Would it be possible to make this optional? Without adding a hard dependency on v7?

Also, @sjudd, we should probably revise the v4 dep and use the more fine-grained exploded libraries (core, fragment, etc).